src/core/CellStructure.hpp

```diff
@@ -56,7 +56,19 @@ enum Resort : unsigned {
   RESORT_LOCAL = 1u,
   RESORT_GLOBAL = 2u
 };
-}
+
+/**
+ * @brief Flags to select particle parts for communication.
+ */
+enum DataPart : unsigned {
+  DATA_PART_NONE = 0u,       /** Nothing */
+  DATA_PART_PROPERTIES = 1u, /** Particle::p */
+  DATA_PART_POSITION = 2u,   /** Particle::r */
+  DATA_PART_MOMENTUM = 8u,   /** Particle::m */
+  DATA_PART_FORCE = 16u,     /** Particle::f */
+  DATA_PART_BONDS = 32u      /** Particle::bonds */
```

Comment:
Docstrings written after the described element require an additional `<` symbol, otherwise they document the element on the next line (I ran doxygen and the enum values did not correspond to their docs).
```suggestion
  DATA_PART_NONE = 0u,       /**< Nothing */
  DATA_PART_PROPERTIES = 1u, /**< Particle::p */
  DATA_PART_POSITION = 2u,   /**< Particle::r */
  DATA_PART_MOMENTUM = 8u,   /**< Particle::m */
  DATA_PART_FORCE = 16u,     /**< Particle::f */
  DATA_PART_BONDS = 32u      /**< Particle::bonds */
```